src/Nox.Lib/Application/Dto/EntityDtoBase.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Nox.Domain;
+using Nox.Types;
+
+namespace Nox.Application.Dto;
+
+/// <summary>
+/// Base Implementation for a Nox EntityDto
+/// </summary>
+public abstract class EntityDtoBase
+{
+    public void TryGetValidationExceptions<T>(string propertyName, Func<T> createAction, Dictionary<string, IEnumerable<string>> validationResult)
```

Comment:
@rochar so for now it's called TryGetValidationExceptions. Please see my other options, let me know if you want to change it:

- CreateAndValidate
- CreateWithValidation
- TryGetValidationExceptions
- TryGetValidationExceptionsOnCreate

Reply:
The TryGetValidationExceptions leads to expecting a bool result and a list of exception, I could suggest 

ExecuteActionAndCollectValidationExceptions , too much verbose but I would prefer this one :)